src/Marten/Storage/StorageFeatures.cs

```diff
@@ -1,245 +1,245 @@
-using System;
```

Comment:
@CodingGorilla was there any change to this file? Is it just some weird EOL formatting issue?

Reply:
This might just have been my R# doing this on it's own as I was making my way through the code.  I think I will just try to recreate these changes on top of master, rather than rebasing.  That is probably easier than rebasing and it will clear up issues like this where I honestly don't remember what I changed.

Reply:
@oskardudycz Yea, originally I didn't do unit tests because I was looking for some guidance as to whether I was doing this "the best way" (as asked in my original comment).  But if this is good, then I'll go ahead and add the tests.

Reply:
For me it looks good. I like that it’s not breaking change and extending what we have. But it would be nice to see what others think. I’ll try to go during the weekend deeper into this feature, but so far so good.